lib/two-fer/analyzer.py

```diff
@@ -49,7 +50,9 @@ def analyze(file_path):
         output['status'] = 'disapprove_with_comment'
         output['comment'] = [malformed_code]
         output['pylint_comment'] = []
-        json_output = json.dumps(output)
+        # [ccare] probably not pythonic but the following hack writes analysis.json into the correct place
+        analysis_out = os.path.dirname(file_path) + '/analysis.json'
+        json_output = json.dumps(analysis_out)
```

Comment:
This just creates a JSON file containing the string where you originally intended to save the JSON, but does not actually change the place it's saved to...